documentation/docs/community/contribute/tips-and-tricks.md

```diff
@@ -83,6 +83,7 @@ poetry install --no-root
 Run live previewer:
 
 ```sh
+cd documentation
```

Comment:
It looks like the commands directly above also include a `cd documentation` command, so this is redundant for people who've just run the installation commands - but this is necessary otherwise.

The way I addressed this challenge in https://github.com/PlanktoScope/PlanktoScope/tree/master/documentation was:

- at the start of the developer instructions, I said: "For the rest of this development guide, we will assume you've cloned it to `/some/path/here/PlanktoScope` - you should replace that with the actual path where your local clone of the PlanktoScope repository is stored on your computer."
- in each block of commands, I included `cd /some/path/here/PlanktoScope/documentation`

Not sure if that's the ideal approach, but that's the precedent right now.